l10n_it_riba_sale_commission/models/account_move.py

```diff
@@ -22,11 +22,98 @@ def _skip_future_payments(self, date_payment_to):
                     max(
                         line.mapped(
                             "distinta_line_ids.riba_line_id.config_id.safety_days"
-                        )
+                        ),
+                        default=0,
                     )
                 )
                 for line in self.invoice_id.line_ids.filtered(lambda r: r.date_maturity)
             ]
             if dates:
                 return date_payment_to < max(dates)
         return super()._skip_future_payments(date_payment_to)
+
+    def _partial_commissions(self, date_payment_to):
```

Comment:
Instead of rewriting the entire method, wouldn't be possible to inherit `_get_reconciled_invoices_partials` and remove the lines that you are excluding here?

Reply:
Hi @renda-dev, Thanks for your review. I inherited the method _get_reconciled_invoices_partials and excluded partial payments directly there.